packages/react-lowcode/src/codegen/tests/list/facade.test.ts

```diff
@@ -23,6 +23,6 @@ describe("codegen facade", () => {
     });
 
     test(".add widget to existing detail page", () => {
-      insertFormWidget({lineNumber: 33,columnNumber: 19, fileName: 'C:\\Private\\mat-app\\lowcode\\packages\\react-lowcode\\src\\codegen\\tests\\detail\\detail-test-file.txt'}, {entityField: getTestProperty('name')[0]})
+      insertFormWidget({lineNumber: 33,columnNumber: 19, fileName: 'C:\\Private\\mat-app\\lowcode\\packages\\react-lowcode\\src\\codegen\\tests\\detail\\detail-test-file.txt'}, {entityField: getTestProperty('test')[0]})
```

Comment:
We really need to remove absolute paths to your local workspace!

Reply:
fixed